osx/certs/README.TXT

```diff
@@ -0,0 +1,2 @@
+Add here Trust Root Certificate
+Warning: Don't put fucking false CA to hack me !
```

Comment:
```suggestion
```

Reply:
Maybe, a day, i could put my own CA servers into this directory, but i could add a .gitignore for .pem or .crt ? It's for a standalone OpenSSL key chain ;)

Reply:
The problem is OSX don't use openSSL but DarwinSSL and i wanted a full standalone OpenSSL Version